src/test/ui/async-await/async-fn-nonsend.stderr

```diff
@@ -12,7 +12,7 @@ note: future is not `Send` as this value is used across an await
   --> $DIR/async-fn-nonsend.rs:24:5
    |
 LL |     let x = non_send();
-   |         - has type `impl std::fmt::Debug`
+   |         - created here
```

Comment:
Yeah, maybe I got a little overzealous here. If the type were written out as you say it wouldn't matter, because auto traits leak. But maybe it's best to guide them down that path anyway..